test/cqlpy/test_secondary_index.py

```diff
@@ -165,6 +165,16 @@ def test_create_index_if_not_exists(cql, test_keyspace):
             cql.execute(f"DROP INDEX {test_keyspace}.abc")
         cql.execute(f"DROP INDEX {test_keyspace}.xyz")
 
+def test_alter_index_if_not_exists(cql, test_keyspace, scylla_only):
```

Comment:
You gave this test a confusing name (and placed it in a confusing place) - the previous test checks CREATE INDEX IF NOT EXISTS. This test doesn't - there is no such thing as ALTER INDEX IF NOT EXISTS. It's just a normal ALTER INDEX on the index doesn't exist :-)